spec/seeds_orders.sql

```diff
@@ -0,0 +1,7 @@
+TRUNCATE TABLE orders RESTART IDENTITY CASCADE; -- replace with your own table name.
```

Comment:
Since the orders have an `item_id` which depends on the records in the `items` table, it might be worth truncating `items` as well to make sure you know which values of `item_id` are valid or not (eg if you run your `ItemRepository` tests first, you might end up creating an extra item or deleting one just before running your `OrderRepository` tests)